### PR TITLE
ci: use FAST_FORWARD_TOKEN in fast-forward.yml

### PR DESCRIPTION
**Pull request by BacLuc**
And check write access and PR status by ourselves,
because the fine grained access token has the permission to merge without the review or checks beeing complete.

I documented it here: https://github.com/ecamp/ecamp3/wiki/deployment
I tested it here: https://github.com/bacluc-test-org/ecamp3/pull/3

### CODE REVIEW

File: .github/workflows/fast-forward.yml [hunk 1]
```diff
@@ -9,19 +9,60 @@ jobs:
...
       (github.event.issue.pull_request != '') &&
-      github.event.comment.body == '/fast-forward' &&
-      (github.event.comment.author_association == 'MEMBER' || github.event.comment.author_association == 'OWNER')
+      github.event.comment.body == '/fast-forward'
     runs-on: ubuntu-latest
     steps:
+      - uses: actions-cool/check-user-permission@v2
+        id: has_permission
+        with:
+          require: 'write'
```
Comment (usu) [inline .github/workflows/fast-forward.yml:18]: Could we also use `'admin'` here? Or is this too strict?
The user `github.actor` is the user who wrote the comment, right?

File: .github/workflows/fast-forward.yml [hunk 2]
```diff
@@ -9,19 +9,60 @@ jobs:
...
+              repo: context.repo.repo,
+            })
+
+      - run: echo $prJSON
+        env:
+          prJSON: ${{ toJSON(fromJSON(steps.current_pr.outputs.result).data) }}
+
+      - name: Comment PR
+        uses: thollander/actions-comment-pull-request@v1
+        if: fromJSON(steps.current_pr.outputs.result).data.mergeable_state != 'clean'
```
Comment (usu) [inline .github/workflows/fast-forward.yml:48]: Did you find any documentation for mergeable_state?

I only found this:
https://github.com/orgs/community/discussions/24299
https://docs.github.com/en/rest/pulls/pulls#get-a-pull-request
https://docs.github.com/en/graphql/reference/enums#mergestatestatus

The last one is from the GraphQL API and has a different property name, but values seem to match mergeable_state form the REST API. So the value is only 'clean' if the PR has no conflicts and all required status checks have passed, right?

File: .github/workflows/fast-forward.yml (hunk 1 above)
Comment (BacLuc) [reply .github/workflows/fast-forward.yml:18]: Could we also use 'admin' here? Or is this too strict?
-> For me no problem, currently all people which need to deploy are admin too

The user github.actor is the user who wrote the comment, right?
-> yes, my tests confirmed that

File: .github/workflows/fast-forward.yml (hunk 2 above)
Comment (BacLuc) [reply .github/workflows/fast-forward.yml:48]: I did not test the case of a conflict, but then the fast forward would anyway fail.
What i tested is: the state is clean if all checks pass and there are enough approving reviews from people with write access.

**Review by usu — APPROVED** [review]

**Review by carlobeltrame — APPROVED** [review]

**Pull request merged**